src/index.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+Hello, world!
```

Comment:
something is missing...
try to type `!` in the empty file and press the `tab` button